addons/website_hr_recruitment/static/tests/tours/website_hr_recruitment.js

```diff
@@ -126,4 +126,28 @@
     },
 ]);
 
+    // This fix addresses an issue that occurred in a website form containing
+    // the 'hide-change-model' attribute. Specifically, when a model-required
+    // field is selected, the alert message should not display an undefined
+    // action name.
+    wTourUtils.registerWebsitePreviewTour('model_required_field_should_have_action_name', {
+        test: true,
+        url: '/jobs',
+    }, [{
+        content: 'Go to the experienced developer job page',
+        trigger: 'iframe a[href="/jobs/experienced-developer-4"]',
+    }, {
+        content: 'Apply for the job',
+        trigger: 'iframe a[href="/jobs/apply/4"]',
+    },
+    ...wTourUtils.clickOnEditAndWaitEditMode(),
+    {
+        content: 'click on the your name field',
+        trigger: 'iframe #hr_recruitment_form div.s_website_form_model_required',
+    }, {
+        content: "Select model-required field",
```

Comment:
Decide on either using single or double quotes. I believe we now go for double quotes when there is no double-quotes inside the string, on the javascript side.